proto/xudrpc.proto

```diff
@@ -229,10 +236,10 @@ message GetOrdersRequest {
   uint32 max_results = 2 [json_name = "max_results"];
 }
 message GetOrdersResponse {
-  // A list of buy orders sorted by descending price
-  repeated Order buy_orders = 1 [json_name = "buy_orders"];
-  // A list of sell orders sorted by ascending price
-  repeated Order sell_orders = 2 [json_name = "sell_orders"];
+  // A list of the orders of peers
```

Comment:
I think I prefer just "A list of peer orders" - it's short and simple but I'm not sure what else would be useful to say.